### PR TITLE
[ADF-2607] Task Form - Number Widget placeholder no longer displayed

### PR DESCRIPTION
**Pull request by camorra-skk**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [x] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)
* https://issues.alfresco.com/jira/browse/ADF-2607

**What is the new behaviour?**
* Removed css to display placeholder after focus on input.

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: lib/core/form/components/widgets/text/text.widget.spec.ts [hunk 1]
```diff
@@ -269,10 +279,18 @@ describe('TextWidgetComponent', () => {
...
             });
 
             it('should show the input mask placeholder', () => {
-                const inputElement: HTMLInputElement = <HTMLInputElement> element.querySelector('#text-id');
+                expect(inputElement).toBeDefined();
+                expect(inputElement).not.toBeNull();
+                expect(inputElement.placeholder).toBe('Phone : (__) ___-___');
+            });
+
+            it('should show the input mask placeholder when clicked', () => {
```
Comment (VitoAlbano) [inline lib/core/form/components/widgets/text/text.widget.spec.ts]: this tests is related to the input mask placeholder that is different from the normal placeholder. Also i don't think the placeholder should be visibile when clicking on the widget.

File: lib/core/form/components/widgets/text/text.widget.ts [hunk 2]
```diff
@@ -39,10 +39,12 @@ export class TextWidgetComponent extends WidgetComponent implements OnInit {
     }
 
     ngOnInit() {
-        if (this.field.params && this.field.params['inputMask']) {
-            this.mask = this.field.params['inputMask'];
+        if (this.field.params) {
             this.placeholder = this.field.params['inputMaskPlaceholder'] ? this.field.params['inputMaskPlaceholder'] : this.field.placeholder;
             this.isMaskReversed = this.field.params['inputMaskReversed'] ? this.field.params['inputMaskReversed'] : false;
+            if (this.field.params['inputMask']) {
```
Comment (VitoAlbano) [inline lib/core/form/components/widgets/text/text.widget.ts]: do we need this if? will it work without?

**Comment by VitoAlbano** [comment]
This will fix the problem only for the text widget, what about the number widget?

File: lib/core/form/components/widgets/text/text.widget.spec.ts [hunk 3]
```diff
@@ -150,6 +150,14 @@ describe('TextWidgetComponent', () => {
                 expect(inputElement.placeholder).toBe('simple palceholder');
             });
 
+            it('should show the field placeholder when clicked', () => {
```
Comment (mauriziovitale) [inline lib/core/form/components/widgets/text/text.widget.spec.ts]: Can you use the async and whenStable method, please?

File: lib/core/form/components/widgets/text/text.widget.spec.ts (hunk 1 above)
Comment (camorra-skk) [reply lib/core/form/components/widgets/text/text.widget.spec.ts]: The similar behaviour is in APS also. When we click on the text area the placeholder value is visible. It disappears only when value is entered in the text area.

File: lib/core/form/components/widgets/text/text.widget.spec.ts (hunk 3 above)
Comment (camorra-skk) [reply lib/core/form/components/widgets/text/text.widget.spec.ts]: The test case is only to check the value of placeholder when clicked. There is no asynchronous call made. So I didnt add async and whenStable methods.

**Pull request merged**